package.json

```diff
@@ -3,15 +3,15 @@
   "type": "module",
   "version": "1.5.0",
   "private": true,
-  "packageManager": "pnpm@8.10.3",
+  "packageManager": "pnpm@9.0.5",
   "description": "Next generation testing framework powered by Vite",
   "engines": {
     "node": "^18.0.0 || >=20.0.0"
   },
   "scripts": {
     "ci": "ni && nr typecheck && nr lint && nr build && nr test:all",
     "ci:docs": "pnpm run build && pnpm run docs:build",
-    "build": "pnpm -r --filter='./packages/**' run build",
+    "build": "pnpm -r --filter @vitest/ui --filter='./packages/**' run build",
```

Comment:
For now I did the same trick from https://github.com/vitest-dev/vitest/pull/5499#issuecomment-2050994092 for the cyclic build issue.

I'm not sure if we should fix something more fundamentally.